github-dark.user.css

```diff
@@ -194,6 +194,7 @@
   } EOT;
 }
 @advanced text font-choice "Code Font" "Menlo"
+@advanced text font-features "Code Font Features" normal
```

Comment:
Should `normal` be quoted?

Reply:
In actual css you don't quote `normal`, but in this instance it looks to work either way and the browser gets `normal` instead of `"normal"`. `base-color` doesn't use quotes, but all the others do. Want me to change them so they're consistent?

Reply:
Yeah, I'd like those values to reflect what goes into the CSS so make them unquoted where they appear unquoted in CSS. I think only `bg-custom` currently violates this. Would appreciate if you could test those variables after the change.